### DISCUSSION TITLE
Too Many Requests Error even though there hasn't been?

### DISCUSSION THREAD

**Discussion by marwan-at-work** [planetscale/discussion#57] [Bug Reports] [answered]
Hi there, I am getting the following error when trying to connect ot the database via planetscale-go:

```
Error 1105: unknown error: couldn't list services found for "us-east-1/***/***/main": the server has received too many requests and has asked us to try again later (get services)
```

It's worth noting that I definitely haven't been trying to connect too many times, though even if I have, I _expect_ that as my service scales horizontally then it certainly will try to connect 10s or 100s of times in parallel so I'd love to see those type of rate limits configured properly. 

Thanks!

**Comment by nickvanw** [comment, marked as answer]
Hey @marwan-at-work! 

The error you're getting is actually due to an internal rate limit that's being hit, not anything to do with your connection volume. As you pointed out, it's normal course of business for a service to initiate 100s of connections (if not thousands) when it's being deployed or scaled out. Our connection handling system doesn't rate limit this when it's functioning properly, but right now it's a bit behind due to some internal scaling limitations.

I'm sorry you're experiencing this, and we're working on getting things back to full capacity as quickly as possible.
